ui/src/main/java/edu/wpi/grip/ui/util/DPIUtility.java

```diff
@@ -13,6 +13,7 @@
     public final static double FONT_SIZE = 11.0 * (isManualHiDPI() ? HIDPI_SCALE : 1.0);
     public final static double SMALL_ICON_SIZE = 16.0 * (isManualHiDPI() ? HIDPI_SCALE : 1.0);
     public final static double LARGE_ICON_SIZE = 48.0 * (isManualHiDPI() ? HIDPI_SCALE : 1.0);
+    public final static double MINI_ICON_SIZE = SMALL_ICON_SIZE / 2.0;
```

Comment:
Why would this be defined relative to `SMALL_ICON_SIZE`?  Just say `8.0 * (isManualHiDPI`...